crates/core_arch/src/x86/sse.rs

```diff
@@ -1607,6 +1611,10 @@ pub unsafe fn _mm_getcsr() -> u32 {
 ///
 ///
 /// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_setcsr)
```

Comment:
I think if this function is going to be deprecated as "impossible to use correctly" its docs should be mostly removed or replaced with some text saying that much. Otherwise, the function's rustdoc page is a small bar saying it's deprecated dwarfed by multiple screens full of text and examples showing what wonderful things it can supposedly be used for.